templates/default/delos.css

```diff
@@ -5910,7 +5910,7 @@ button.close {
   height: 10px;
   margin: 1px;
   text-indent: -999px;
-  border: 1px solid #ffffff;
+  border: 1px solid #fff;
```

Comment:
Yes, this is a part of the install/build-process-problem. The picking of the line would (hopefully) not have to be done, if we'd use the same options (versions, whatever matters) for lessc. I agree that it would be cumbersome to pick lines manually. On the other hand: calling blame on some of the `background-color: #fff` lines would now show your commit with the message "Less for Glyph and Counter refactored. Variables added.", that is why i suggested picking the line in the first place. *sigh*

So don't mind for this PR, but at some point we definitely need to fix this, as this messes with our mental sanity.